src/components/Bullets.js

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { PALETTE } from "../shared";
+
+const ListEl = styled.li`
+  position: relative;
+  margin: 0.5rem 0;
+
+  &::before {
+    content: "";
+    position: absolute;
+    left: -1.4rem;
+    top: 0.7rem;
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background-color: ${PALETTE.black60};
+  }
+
+  &:first-child > ul > & {
+    &::before {
+      width: 4px;
+      height: 4px;
+      border: 1px solid ${PALETTE.black60};
+      background-color: transparent;
+    }
+  }
+
+  &:nth-child(2) > ul > & {
+    &::before {
+      width: 6px;
+      height: 6px;
+      border-radius: initial;
+      background-color: ${PALETTE.black60};
+    }
+  }
+`;
+
+const El = styled.ul`
+  position: relative;
+  list-style-type: none;
+  color: ${PALETTE.black60};
+  padding-left: 1.375rem;
+`;
+
+/**
+ * Note: This exports a React component instead of a styled-component.
+ * [Design Mockup](https://zpl.io/2ZXQ0YJ)
+ */
+
+const Bullets = ({ data }) => (
```

Comment:
I try to name elements in a way that precisely describes what the component does; it makes things easier to understand. When I see the name "Bullets", it makes me think it's a series of bullets in a row, or it's the component for the bullet glyph, versus something more specific like "BulletedList" or "UnorderedList" or whatever.